modules/querier/querier.go

```diff
@@ -327,6 +333,14 @@ func (q *Querier) SearchTagValues(ctx context.Context, req *tempopb.SearchTagVal
 		return nil, errors.Wrap(err, "error extracting org id in Querier.SearchTagValues")
 	}
 
+	// Hard-coded values?
+	staticValues := search.GetStaticTagValues(req.TagName)
+	if len(staticValues) > 0 {
+		return &tempopb.SearchTagValuesResponse{
+			TagValues: staticValues,
+		}, nil
+	}
```

Comment:
Instead of returning early, what do you think about still querying the ingesters and combining both lists?
I can imagine a case in which someone uses the `error` tag to capture the error message (which is perfectly fine since this is not reserved in OTel). In this case we should still show all tag values for `error` imo.

Reply:
I did originally implement it to call the ingesters and combine both lists (like `SearchTags`), but in the case of `status.code` it seemed better to do a full value replacement (else the autocomplete lists also contains '0', '2'...), and hence the ingester calls were unnecessary and it moved to exit early.  We could split the functionality up where some tags exit early and some merge (maybe names like `GetStaticTagValues` and `GetExtraTagValues`), or just have `status.code` merge and show all values in the autocomplete. Thoughts?

Reply:
Oh yeah, I didn't think about the `status.code` tag.

I'm favouring returning both the actual values (`0`, `1`, `2`) and the virtual values (`unset`, `ok`,...) because that is most transparent:
- the trace view currently shows `span.status` with their numeric value, so people might be more used to seeing these.
- if we don't show the labels, they might not know it's possible to use these as well.

I just hope seeing both numbers and labels isn't confusing: will it be clear that they are aliases of each other?

Reply:
Updated to merge all virtual values according to discussion. Even though the list will contain all values and might be a little confusing, it is also the most transparent and flexible.